### DISCUSSION TITLE
Problem with scatter diagram - scatter diagram changes values

### DISCUSSION THREAD

**Discussion by greysquirrel-coding** [eclipse-birt/birt#782] [Q&A] [answered]
I hope someone can help me. I have a problem with a diagram in BIRT. I want to make a scatter diagram with the positions of extrasolar planetsystems in the sky. The right ascension (x-axis) has values between 0 and 360 degrees and the declination (y-axis) has values between -90 and 90 degrees. Somehow the scatter diagram changes the values of the y-axis. I have checked the values in the database. No value for the y-axis has more than 90 degrees or less than -90 degrees. Does anyone know a solution? I have tried to solve it with a line diagram by hiding the line, but the result is the same
![ra dec data example](https://user-images.githubusercontent.com/79271603/148298394-3a3f46d8-2460-404f-aa56-ae254eb1da4b.PNG)
![ra dec diagram birt](https://user-images.githubusercontent.com/79271603/148298434-03ffcfb6-9186-487d-965d-93face7e62e7.PNG)
![ra dec min max](https://user-images.githubusercontent.com/79271603/148298444-d049e2f3-c9b5-49b0-9293-96883f51bceb.PNG)
.

**Comment by wimjongman** [comment, marked as answer]
Could it be a conversion problem? E.g. are the decimal positions correct, float to int, etc..

Maybe you can create a table from the datasource and check the values as BIRT sees them.

**Reply by greysquirrel-coding** [reply]
I have solved the problem. I had to disable the grouping of the data.
![disable grouping](https://user-images.githubusercontent.com/79271603/148411627-60b2eeaf-1c08-4b41-a097-c963266e3a36.PNG)
![works fine](https://user-images.githubusercontent.com/79271603/148411667-a8446164-64f0-4d60-8212-7dd304bc7ccd.PNG)

**Reply by wimjongman** [reply]
Looks great!
